docs/whats-new/dotnet-docs-2022-06-01.md

```diff
@@ -0,0 +1,80 @@
+---
+title: ".NET docs: What's new for June 2022"
+description: "What's new in the .NET docs for June 2022."
+ms.date: 07/01/2022
+---
+
+# .NET docs: What's new for June 2022
+
+Welcome to what's new in the .NET docs for June 2022. This article lists some of the major changes to docs during this period.
+
+## .NET Framework
+
+### New articles
+
+- [SqlMoney.ToSqlInternalRepresentation Method](../framework/additional-apis/system.data.sqltypes.sqlmoney.tosqlinternalrepresentation.md)
+
+## .NET fundamentals
+
+### New articles
+
+- [Kestrel: Default HTTPS binding removed](../core/compatibility/aspnet-core/7.0/https-binding-kestrel.md)
```

Comment:
Might be an idea to have a separate breaking changes section and put anything under docs/core/compatibility in there.

Reply:
If you want to make a new PR adding a section for that [here](https://github.com/dotnet/docs/blob/main/.whatsnew.json#L45-L48). It should just work. The new section would have to come before the "core" section.